lib/ex_debug_toolbar.ex

```diff
@@ -145,9 +166,9 @@ defmodule ExDebugToolbar do
   defmacro pry do
     code_snippet = Breakpoint.code_snippet(__CALLER__)
     quote do
-      BreakpointRepo.insert(%Breakpoint{
-        id: System.unique_integer |> to_string,
-        pid: self(),
+      {:ok, %{uuid: request_id}} = ExDebugToolbar.get_request()
```

Comment:
you know, this bugs me. you access request twice: here and later on when adding this breakpoint to it. that's because we break encapsulation of data collection. I see two options :

* proceed and acknowledge that we break it in protocol: either pass full request or additionally request id for now
* revert to mnesia match spec option by breakpoint id only

Reply:
mmm, let me sleep on this. but it seems that the mnesia match is the way to go for now, until we have more usecases for this
I'm trying to think, if we kept ajax requests in side a request, we would have
```
[
   request_1 -> [ajax_1_id, ajax_2_id],
   ajax_1,
   ajax_2,
]
```
in that case, we already have requests ids, which are global. It would be the same case as breakpoint id being global